templates/nanoagent.yaml

```diff
@@ -47,9 +48,11 @@ spec:
         - name: spyderbat-nanoagent
           image: {{ .Values.image.registry}}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
           imagePullPolicy: {{ .Values.image.pullpolicy }}
+{{- if .Values.aws.secretsmanager.enabled }}
+          command: ["/bin/sh","-c"]
```

Comment:
bash might be better then sh here.     Bash is available and that's what most/all of the testing has been with.